### PR TITLE
[Geolocation] add alwaysRecenterToPosition property

### PR DESCRIPTION
**Pull request by candux**
I find the current Geolocation component unintuitive. By default,
it will recenter to the current position every second on my smartphone.

If noCenterAfterDrag is set, it will
- center to the current position if the map hasn't been dragged yet
- don't center to the position if the map has been dragged. Even when
  you disable the component and enable it again.

Services like Google Maps center once to the current position and
then every time you press the the location-button again.
I think most users expect such behavior.

I therefor introduced a new property alwaysRecenterToPosition.
By default or if noCenterAfterDrag is set, the behavior stays the
same.
If alwaysRecenterToPosition is set to false, the map will recenter to
the current position and every time the component is enabled again.

Also rename isCentered to recenterToPostion

### CODE REVIEW

**Comment by oterral** [comment]
I guess the google behavior should be the default one.

**Comment by candux** [comment]
Making it the only option would make the logic easier. But you would have to change the property name at the same time in trafimage

File: src/components/Geolocation/Geolocation.js [hunk 1]
```diff
@@ -66,10 +72,10 @@ class Geolocation extends PureComponent {
       source: new VectorSource(),
     });
 
-    this.isCentered = true;
+    this.recenterToPostion = true;
```
Comment (oterral) [inline src/components/Geolocation/Geolocation.js:75]: Can we rename this property to `isRecenteringToPosition`  otherwise we could think it's a function.

**Comment by oterral** [comment]
let's merge #548 before this one

File: src/components/Geolocation/Geolocation.js (hunk 1 above)
Comment (candux) [reply src/components/Geolocation/Geolocation.js]: sure

**Comment by oterral** [comment]
@candux conflict !!!!!

**Comment by candux** [comment]
conflict fixed

**Comment by oterral** [comment]
@candux thx

**Pull request merged**